README.md

```diff
@@ -18,6 +18,82 @@ to facilitate unit-testing of OpenTracing Java instrumentation.
 
 Packages are deployed to Maven Central under the `io.opentracing` group.
 
+## Usage
+
+### Initialization
+
+Initialization is OpenTracing-implementation-specific. Generally speaking, the pattern is to initialize a `Tracer` once
+for the entire process and to use that `Tracer` for the remainder of the process lifetime. The
+[GlobalTracer](https://github.com/opentracing-contrib/java-globaltracer) repository provides a helper for singleton
+access to the `Tracer` as well as `ServiceLoader` support for OpenTracing Java implementations.
+
+### Span Scheduling and the "active" Span
```

Comment:
REVIEWERS: the text from here to the end of the README summarizes the high order user-facing bits